### DISCUSSION TITLE
Node Red debug Query

### DISCUSSION THREAD

**Discussion by Fiorello8** [alexandrainst/node-red-contrib-postgresql#40] [Q&A]
Hi 

Is it possibile debug the query using  nodered ?

**Comment by Alkarex** [comment]
I would recommend testing your queries directly in your database client, e.g. psql or pgAdmin, before deploying them in Node-RED.
However, this extension should provide you some usable error messages (cf. https://github.com/alexandrainst/node-red-contrib-postgresql/discussions/35#discussioncomment-5253173 )
